### PR TITLE
Add option to disable graceful shutdown

### PR DESCRIPTION
**Pull request by davidpdrsn**
### Checklist

* [x] I have read the [Contributor Guide](../../CONTRIBUTING.md)
* [x] I have read and agree to the [Code of Conduct](../../CODE_OF_CONDUCT.md)
* [x] I have added a description of my changes and why I'd like them included in the section below

### Description of Changes

I've found this useful during local development. So the server shuts down right away.

### CODE REVIEW

**Comment by repi** [comment]
Could we call it "disgraceful shutdown"? 😄

File: src/server.rs [hunk 1]
```diff
@@ -437,12 +444,20 @@ impl<F, H> Server<F, H> {
...
+    /// This is useful during tests or local development.
     pub fn disable_health_and_metrics(mut self, disable_health_and_metrics: bool) -> Self {
         self.disable_health_and_metrics = disable_health_and_metrics;
         self
     }
 
+    /// Disable graceful shutdown.
+    ///
+    /// This is useful during tests or local development.
+    pub fn disable_graceful_shutdown(mut self, disable_graceful_shutdown: bool) -> Self {
```
Comment (Nehliin) [inline src/server.rs:456]: Do we need to take a bool as argument if it's called "disable"? Perhaps `use_graceful_shutdown` makes more sense if we want to take a bool as argument. But I guess in that case we should probably change the health and metrics as well. Anyway not something important just something I came to think about.

**Review by Nehliin — APPROVED** [review]
Thanks!

File: src/server.rs (hunk 1 above)
Comment (davidpdrsn) [reply src/server.rs]: Good point! I've renamed it to `with_graceful_shutdown` and `with_health_and_metrics`.

**Review by Nehliin — APPROVED** [review]

**Pull request merged**